components/userDashboard/application.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import TabsWithTable from "../../components/basic/TabsWithTable";
+
+const Application = ({ user }) => {
+  const [applications, setApplications] = useState([]);
+  const [error, setError] = useState("");
+  const userEmail = user.email;
+
+  const fetchApplications = async () => {
+    try {
+      setError("");
+      const response = await axios.get(
+        `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/jobApplication/email/${userEmail}`,
```

Comment:
we use cookies, user info is there. not require to send the user email. route can be fetch application only like jobApplication/fetch